types/index.d.ts

```diff
@@ -180,4 +183,22 @@ interface IdamDetails {
   family_name: string;
 }
 
-type Middleware = (req: Express.Request, res: Express.Response, next: any) => void;
```

Comment:
why deleting this?

Reply:
Hasn't really been deleted just moved to line 203. 
After solving the conflicts, it thinks it was deleted.  But it's still there on line 203